ci/jenkins/nephe-gc.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+_usage="Usage: $0 [--goVcPassword <Password for VC>]
+  --goVcPassword          Password to the user name for VC.
+  --terraform-dir         directory for terraform."
+function echoerr() {
+  >&2 echo "$@"
+}
+
+function print_usage() {
```

Comment:
where are you using print_usage and print_help. Please update the argument parser

Reply:
done